tests/ott_spec.py

```diff
@@ -19,19 +22,26 @@ def on_start(self):
         return None
 
     def on_stop(self, telemetry, stdout: str, stderr: str, returncode: int) -> None:
-        attributes = extract_leaves(telemetry, "trace_requests", "pbreq", "resource_spans", "resource", "attributes")
+        assert_traces_to_spec(telemetry)
+        assert_metrics_to_spec(telemetry)
 
-        assert get_attribute(attributes, "telemetry.sdk.name")
-        assert get_attribute(attributes, "telemetry.sdk.version")
-        assert get_attribute(attributes, "telemetry.sdk.language")
+    def is_http(self):
+        return False
 
-        assert get_attribute_str(attributes, "telemetry.distro.version")
-        assert get_attribute_str(attributes, "telemetry.distro.name") == "splunk-opentelemetry"
 
-        assert get_attribute(attributes, "process.pid")
+def assert_traces_to_spec(telemetry):
+    attrs = extract_leaves(telemetry, "trace_requests", "pbreq", "resource_spans", "resource", "attributes")
+    assert get_attribute(attrs, "telemetry.sdk.name")
+    assert get_attribute(attrs, "telemetry.sdk.version")
+    assert get_attribute(attrs, "telemetry.sdk.language")
+    assert get_attribute_str(attrs, "telemetry.distro.version")
+    assert get_attribute_str(attrs, "telemetry.distro.name") == "splunk-opentelemetry"
+    assert get_attribute(attrs, "process.pid")
```

Comment:
IMO this is not necessary. See https://github.com/signalfx/splunk-otel-python/issues/557#issuecomment-2513780042

Reply:
Thx. I'll move this to another test that's not about the spec.

Reply:
> `process.pid` is in [the spec](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.18.0/specification/resource/semantic_conventions/process.md#process), so maybe we should keep it?

You are right! This one is important. There is something which takes advantage of it (this was the reason it was added it to the GDI spec).